src/reanimated2/interpolation.js

```diff
@@ -51,3 +52,24 @@ export function interpolate(x, input, output, type) {
   }
   return internalInterpolate.apply({}, [x].concat(narrowedInput).concat(type));
 }
+
+export function interpolateColor(x, input, output, type) {
+  'worklet';
+
+  // Extrapolate.CLAMP is the default interpolation type for colors
+  type = type || Extrapolate.CLAMP;
+
+  const startColor = destructColor.apply({}, output[0]);
+  const endColor = destructColor.appy({}, output[1]);
+
+  if (startColor === undefined || endColor === undefined) {
+    return undefined;
+  }
+
+  const r = interpolate.apply({}, x, input, [startColor[0], endColor[0]], type);
```

Comment:
shouldn't those args be wrapped in an array like `const r = interpolate.apply({}, [x, input, [startColor[0], endColor[0]], type]);` ?

Reply:
here and for the 3 following calls

Reply:
@likern, any update on this?